src/anyvar/storage/postgres.py

```diff
@@ -127,6 +127,10 @@ def __delitem__(self, name: str) -> None:
             cur.execute("DELETE FROM vrs_objects WHERE vrs_id = %s;", [name])
         self.conn.commit()
 
+    def wait_for_writes(self):
+        """Return true once any currently pending database modifications have been completed."""
+        pass
```

Comment:
I don't think the pass is necessary. Is there an existing issue to do this for postgres backend? 

```suggestion
```

Reply:
IMO we shouldn't worry about implementing it PG-side until there's a use case/demand for doing something like queueing. I think there's a world in which it could be preferable to retain the PG implementation as a more streamlined read/write option.

That said, it might be good to note in the docstring that this method is just a stub to retain compatibility with the base class, but shouldn't do anything unless we decide to implement queueing logic for the postgres writer.

Reply:
Then I don't think it should be an abstractmethod in the `_Storage` class and should only exist in the snowflake backend

Reply:
I think the `wait_for_writes()` method should remain on `_Storage` because it is called from `anyvar.restapi.main` which does not understand which backend it is using.  The REST API method implementations only calls methods on the `_Storage` base class.  Without it on the base class, calling `/vcf` with `allow_async_write=no` results in a 500 error if the backend store in Postgres.

I cleaned up the comments and removed the `pass` line.

Reply:
@ehclark that makes sense. Thanks